client/js/table-view.js

```diff
@@ -59,7 +61,7 @@ class TableView {
 				const position = {col: col, row: row};
 				const value = this.model.getValue(position);
 				const td = createTD(value);
-				
+				td.id = col + ', ' + row;
```

Comment:
This is a perfect spot to use a template literal ( ` ` ) rather than string interpolation.